src/gt4py/backend/gtc_backend/cuda/backend.py

```diff
@@ -53,6 +54,7 @@ def __call__(self, definition_ir) -> Dict[str, Dict[str, str]]:
             "oir_pipeline", DefaultPipeline(skip=[NoFieldAccessPruning])
         )
         oir = oir_pipeline.run(base_oir)
+        oir = FillFlushToLocalKCaches().visit(oir)
```

Comment:
Thats the non-overengineered way of doing it :) And I was talking about changing the pipeline infrastructure :man_facepalming: